### PR TITLE
Validation numbers

### PR DESCRIPTION
**Pull request by tgandrews**
### What is the context of this PR?
This adds validation numbers to the left hand nav. It does nothing with isNew and so currently shows all errors.

### How to review 
1. Create a questionnaire with validation errors
1. See the error count reflect in the left hand navigation
1. See the changes in validation count updated in real time

### Dependencies
- [x] - https://github.com/ONSdigital/eq-author-terraform/pull/22

### CODE REVIEW

File: eq-author/package.json [hunk 1]
```diff
@@ -87,6 +87,7 @@
     "apollo-link-context": "latest",
     "apollo-link-error": "latest",
     "apollo-link-http": "latest",
+    "apollo-link-ws": "^1.0.18",
```
Comment (pricea1) [inline eq-author/package.json]: Should this be "latest"?

File: eq-author-api/schema/resolvers/withWritePermission.js [hunk 2]
```diff
@@ -18,6 +19,10 @@ const withWritePermission = mutation => async (root, args, ctx) => {
   const result = await mutation(root, args, ctx);
   await saveQuestionnaire(ctx.questionnaire);
   ctx.validationErrorInfo = validateQuestionnaire(ctx.questionnaire);
+  pubsub.publish("validationUpdated", {
```
Comment (SamGodwin2) [inline eq-author-api/schema/resolvers/withWritePermission.js]: Could we push this down into save questionnaire, seems pretty unrelated to WritePermission and I don't think we ever want to call save and not validate?

File: eq-author-api/schema/tests/page.test.js [hunk 3]
```diff
@@ -336,4 +336,69 @@ describe("page", () => {
...
+                    ],
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      });
+
+      questionnaire = ctx.questionnaire;
```
Comment (SamGodwin2) [inline eq-author-api/schema/tests/page.test.js]: This line and 353 aren't used.

Comment (tgandrews) [reply eq-author-api/schema/tests/page.test.js]: They delete the query at the end.

File: eq-author-api/schema/resolvers/withWritePermission.js (hunk 2 above)
Comment (tgandrews) [reply eq-author-api/schema/resolvers/withWritePermission.js]: Now renamed to `createMutation` so this wrapper no longer exists

**Comment by tgandrews** [comment]
[Validator - number notifications (8)](https://trello.com/c/8ynDssd8/1066-validator-number-notifications-8)

**Comment by tgandrews** [comment]
@pricea1 please re-review as there have been quite a few changes after the public private and additional option validation
